packages/cli/src/commands/setup/pm2/templates/ecosystem.config.js.template

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
So a few things to discuss on this one:

**A) Running web and api as separate processes doesn't work - graphql endpoint is unreachable**
I'm not sure exactly why... but I suspect I originally setup serve web with the flag `--apiPath` to bbe able to automatically proxy and forward requests to a different server. 

**B) Why not run both servers**
Changing the setup to run `yarn rw serve` without any arguments works correctly. Is there a reason we shouldn't make this the default?

**C) Should we set the instances to `max` instead of leaving it at 2?**

Reply:
Hey @dac09! What do you think of @cannikin suggestion in https://github.com/redwoodjs/redwood/pull/3650#issuecomment-1014949334?